### PR TITLE
logging cleanup (see issue #526)

### PR DESCRIPTION
**Pull request by bartnijssen**
* Removed function names from logging messages
* Removed newlines from the end of logging messages (added by log_*() functions)

### CODE REVIEW

**Comment by bartnijssen** [comment]
@jhamman : I simplified the logging in some of the root_brent a bit, so look at that in particular. If you merge this then also go ahead and close the issue.

**Comment by bartnijssen** [comment]
@jhamman: When you merge the PR just squash the commits. Some back-and-forth merges that didn't actually do anything.

File: vic/vic_run/src/calc_atmos_energy_bal.c [hunk 1]
```diff
@@ -199,23 +199,20 @@ error_print_atmos_energy_bal(double  Tcanopy,
...
+             "Tcanopy = %f\n"
+             "LatentHeat = %f\n"
+             "NetRadiation = %f\n"
+             "Ra = %f\n"
+             "Tair = %f\n"
+             "atmos_density = %f\n"
+             "InSensible = %f\n"
+             "*SensibleHeat = %f\n"
+             "Try increasing CANOPY_DT to get model to complete cell.\n"
+             "Then check output for instabilities.\n",
```
Comment (jhamman) [inline vic/vic_run/src/calc_atmos_energy_bal.c]: extra newline character here.

File: vic/vic_run/src/calc_atmos_energy_bal.c [hunk 2]
```diff
@@ -284,19 +281,18 @@ error_print_atmos_moist_bal(double  VPcanopy,
...
-    fprintf(LOG_DEST, "vp = %f\n", vp);
-    fprintf(LOG_DEST, "AtmosLatent = %f\n", *AtmosLatent);
-
-    log_err("Finished writing calc_atmos_moist_bal variables.\nTry increasing "
-            "CANOPY_VP to get model to complete cell.\nThen check output for "
-            "instabilities.");
+    log_err("VPcanopy = %f\n"
+             "InLatent = %f\n"
+             "Lv = %f\n",
+             "Ra = %f\n",
```
Comment (jhamman) [inline vic/vic_run/src/calc_atmos_energy_bal.c]: what do these commas do?

File: vic/vic_run/src/compute_derived_lake_dimensions.c [hunk 3]
```diff
@@ -85,19 +85,19 @@ compute_derived_lake_dimensions(lake_var_struct *lake,
...
     lake->sarea = lake->surface[0];
     status = get_volume(lake_con, lake->ldepth, &tmp_volume);
     if (status < 0) {
-        log_err("Error in get_volume: record = %d, depth = %f, "
+        log_err("Error: record = %d, depth = %f, "
                 "volume = %e", 0, depth, tmp_volume);
     }
     else if (status > 0) {
-        log_err("Warning in get_volume: lake depth exceeds maximum; "
+        log_err("Warning: lake depth exceeds maximum; "
```
Comment (jhamman) [inline vic/vic_run/src/compute_derived_lake_dimensions.c]: these two are confusing.  I would just drop the Error/Warning portions of these strings.

Comment (jhamman) [inline vic/vic_run/src/compute_derived_lake_dimensions.c]: at a minimum, the `log_err("Warning...` should go.

File: vic/vic_run/src/calc_atmos_energy_bal.c (hunk 1 above)
Comment (bartnijssen) [inline vic/vic_run/src/calc_atmos_energy_bal.c]: Fixed

**Comment by jhamman** [comment]
The root brent logic looks good to me.  Just a few nits before I merge.

File: vic/vic_run/src/calc_atmos_energy_bal.c (hunk 2 above)
Comment (bartnijssen) [inline vic/vic_run/src/calc_atmos_energy_bal.c]: Nothing - they should not be there - leftover from previous implementation. Fixed

File: vic/vic_run/src/compute_derived_lake_dimensions.c (hunk 3 above)
Comment (bartnijssen) [inline vic/vic_run/src/compute_derived_lake_dimensions.c]: Agreed - fixed

**Pull request merged**